src/tracker-component.js

```diff
@@ -281,6 +281,19 @@ export const Tracker = (config? : Config = defaultTrackerConfig) => {
         console.log('PayPal Shopping: Safari trackers enabled.');
     }
 
+    // TODO: Make paypal url dynamic to env
+    const clientId = getClientID();
+    const merchantId = getMerchantID()[0];
+    const savedPropertyId = window.localStorage.getItem(`${ clientId }-${ merchantId }`);
+    const propertyId = savedPropertyId
+        ? savedPropertyId
+        : fetch(`https://paypal.com/tagmanager/containers/xo?url=${ urcurrentUrl }&mrid=${ merchantId }`)
+            .then(container => {
+                window.localStorage.setItem(`${ clientId }-${ merchantId }`, container.id);
```

Comment:
Can you add propertyId as part of the localStorage item name so we know what it is at a glance?